ubuntu_bridge_network_configuration.sh

```diff
@@ -10,12 +10,18 @@ source lib/common.sh
 if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
      # Adding an IP address in the libvirt definition for this network results in
      # dnsmasq being run, we don't want that as we have our own dnsmasq, so set
-     # the IP address here
+     # the IP address here.
+     # Create a veth iterface peer.
+     sudo ip link add ironicendpoint type veth peer name veth2
```

Comment:
Can you please rename veth2 to something else ? like ironicendpoint-veth2 or anything that would not be as predictible for collisions. thank you

Reply:
Renamed. It is not really explicit or not ironicendpoint-veth2 because maximum length of characters should be 16 for the interface name.